src/components/CallView/Grid/Grid.vue

```diff
@@ -41,7 +43,9 @@
 					<button v-if="hasPreviousPage && gridWidth > 0 && isStripe && showVideoOverlay"
 						class="grid-navigation grid-navigation__previous"
 						@click="handleClickPrevious">
-						<ChevronLeft decorative
+						<ChevronLeft
+							decorative
```

Comment:
As [the icon will be now hidden from screen readers](https://github.com/robcresswell/vue-material-design-icons#props) an `aria-label` should be added to the parent button. Or a tooltip. But there are other cases in which this should be done too, so I think it would be better to solve it in another pull request (maybe as part of #4621 once rebased?)

Reply:
I'll do it in this PR right away.

Also I think aria-label needs to not describe the icon like the title did (what does a "chevron icon" do anyway) but the action that the button does.

Reply:
> Also I think aria-label needs to not describe the icon like the title did (what does a "chevron icon" do anyway) but the action that the button does.

Oh, totally. But my point was that without `decorative` a screen reader would say (as far as I understand, I have not tried it) _Chevron Icon Left_ which, even if it is not the best description, at least says _icon left_, but with `decorative` and without aria-label or tooltip on the button then the screen reader would only say _button_, which is even less descriptive.

Reply:
indeed